lab08-kaylee/README.md

```diff
@@ -0,0 +1,44 @@
+## Lab-08: Single-Resource API
+
+## About This Project
+
+* Build an HTTP server using the 'http' Node module.
+* Create an Object constructor used to produce instances of a resource.
+* Create a Router constructor to manage GET, POST, PUT and DELETE requests to the server.
+
+## Project Dependencies
+
+* chai, chai-http, debug and uuid
+* To install:
+  * npm chai
+  * npm chai-http
+  * npm debug
+  * npm uuid
+
+## Developer Dependencies
+
+* mocha
+* To install:
+  * npm mocha
+
+## Making Requests
+
+* Example GET request
+  * In terminal (assuming httPie installed): http get :3000/api/note name="todo" date=74
+  * Expected output:
+
+* Example POST request
+  * In terminal (assuming httPie installed): http post :3000/api/note name="todo" date=75
+  * Expected output:
+
+* Example PUT request
+  * In terminal (assuming httPie installed): http post :3000/api/note name="todo" date=75
+  * Expected output:
+
+* Example DELETE request
+  * In terminal (assuming httPie installed): http post :3000/api/note name="todo" date=75
```

Comment:
might wanna rename your httpie "crud operations" in your terminal examples.  it's a bit misleading making post a delete, put from a delete, etc.